frontend/www/js/omegaup/arena/ranking.ts

```diff
@@ -114,23 +175,21 @@ export function onRankingChanged({
       if (
         problems[alias] &&
         rank.username === currentUsername &&
-        problems[alias].languages.length > 0
+        problems[alias].acceptsSubmissions
       ) {
-        const currentPoints = problems[alias].points;
-
-        const currentProblem = navbarProblems.find(
+        const currentPoints = rank.problems.filter(
           (problem) => problem.alias === alias,
-        );
+        )[0].points;
```

Comment:
```suggestion
        const currentPoints = rank.problems.find(
          (problem) => problem.alias === alias,
        ).points;
```

me gustaba más el patrón usado antes. si se queja por la nulabilidad del resultado, hay que manejarla en vez de sacarle la vuelta usando `filter`.

también, cuál es la diferencia entre `currentPoints` y `problem.points`? por qué L190 usa `problem.points` y L192 usa `currentPoints`?

Reply:
Ah, los dos salen de donde mismo. Quizás esto ya no sea necesario. Gracias por la observación.

Bueno, le voy a dar una revisada a `arena.ts` para ver cuál era el propósito, para ver si es necesario agregar esto.

Reply:
Parece que arena sólo lo declara y no lo vuelve a usar nunca. Así que aquí de plano ya ni lo vamos a agregar.